pkgs/misc/drivers/utsushi/default.nix

```diff
@@ -0,0 +1,154 @@
+{stdenv, fetchurl, autoreconfHook, boost, gtkmm2,
+pkg-config, libtool, udev, libjpeg, file, texlive,
+libusb, libtiff, imagemagick, sane-backends, tesseract,
+}:
+
+/*
+Alternatively, this package could use the "community source" at 
+https://gitlab.com/utsushi/utsushi/
+Epson provides proprietary plugins for networking, ocr and some more
+scanner models. Those are not (yet ?) packaged here.
```

Comment:
Would be easier to upstream these patches to a community fork.

Reply:
I was unable to build the community sources (autoconf fails with
```
configure.ac:54: error: possibly undefined macro: AS_IF
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:205: error: possibly undefined macro: AC_MSG_ERROR
```
)

Reply:
https://gitlab.com/utsushi/utsushi/issues/81